README.md

```diff
@@ -77,3 +78,7 @@ print(unit_alias('kft.lbf'))
 
 If you have suggestions for improvement or you found bugs,
 please don't hesitate to put them in the issue list.
+
+
+## License
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/Schlumberger/UOM/pullRequest?prid=9389293)
```suggestion
## License

```